osfoffline/views/rsc/preferences_rc.py

```diff
@@ -44,32 +44,33 @@ def setupUi(self, Preferences):
         self.tab_2 = QtWidgets.QWidget()
         self.tab_2.setObjectName("tab_2")
         self.groupBox_2 = QtWidgets.QGroupBox(self.tab_2)
-        self.groupBox_2.setGeometry(QtCore.QRect(10, 10, 581, 51))
+        self.groupBox_2.setGeometry(QtCore.QRect(10, 0, 541, 51))
         self.groupBox_2.setObjectName("groupBox_2")
-        self.groupBox_3 = QtWidgets.QGroupBox(self.groupBox_2)
-        self.groupBox_3.setGeometry(QtCore.QRect(280, 50, 561, 51))
-        self.groupBox_3.setObjectName("groupBox_3")
         self.label = QtWidgets.QLabel(self.groupBox_2)
-        self.label.setGeometry(QtCore.QRect(30, 16, 411, 31))
+        self.label.setGeometry(QtCore.QRect(20, 20, 411, 31))
         self.label.setObjectName("label")
         self.accountLogOutButton = QtWidgets.QPushButton(self.groupBox_2)
-        self.accountLogOutButton.setGeometry(QtCore.QRect(440, 10, 99, 31))
+        self.accountLogOutButton.setGeometry(QtCore.QRect(440, 20, 99, 31))
         self.accountLogOutButton.setObjectName("accountLogOutButton")
         self.groupBox_4 = QtWidgets.QGroupBox(self.tab_2)
-        self.groupBox_4.setGeometry(QtCore.QRect(10, 70, 561, 211))
+        self.groupBox_4.setGeometry(QtCore.QRect(10, 60, 541, 241))
         self.groupBox_4.setObjectName("groupBox_4")
         self.groupBox_5 = QtWidgets.QGroupBox(self.groupBox_4)
-        self.groupBox_5.setGeometry(QtCore.QRect(10, 30, 541, 161))
+        self.groupBox_5.setGeometry(QtCore.QRect(10, 20, 521, 211))
         self.groupBox_5.setObjectName("groupBox_5")
         self.changeFolderButton_2 = QtWidgets.QPushButton(self.groupBox_5)
-        self.changeFolderButton_2.setGeometry(QtCore.QRect(430, 50, 99, 31))
+        self.changeFolderButton_2.setGeometry(QtCore.QRect(420, 100, 99, 31))
         self.changeFolderButton_2.setObjectName("changeFolderButton_2")
         self.treeWidget = QtWidgets.QTreeWidget(self.groupBox_5)
-        self.treeWidget.setGeometry(QtCore.QRect(10, 20, 311, 131))
+        self.treeWidget.setGeometry(QtCore.QRect(10, 50, 311, 151))
```

Comment:
The setting here doesn't look nice in windows system which actually the two buttons are stick to the top of the syn menu box. I will make some number change when I merge to make it looks better.